src/components/Login.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import LoginLocal from './LoginLocal'
+import RegisterNewUser from './RegisterNewUser'
+export default class Login extends React.Component{
```

Comment:
+1! I found the [Usage with React](http://redux.js.org/docs/basics/UsageWithReact.html) section of the Redux docs to be a helpful place to look for nicely written es6 React if you want multiple examples